edocumentation-domain/src/main/java/org/dcsa/edocumentation/domain/persistence/entity/Booking.java

```diff
@@ -63,19 +65,37 @@
 @Entity
 @Table(name = "booking")
 @BookingValidation(groups = AsyncShipperProvidedDataValidation.class)
-public class Booking extends AbstractStateMachine<BkgDocumentStatus> implements Persistable<UUID> {
-
-
-  private static final Set<BkgDocumentStatus> CAN_BE_VALIDATED =
-    Set.of(BkgDocumentStatus.RECE, BkgDocumentStatus.PENC);
-
-
-  private static final DFADefinition<BkgDocumentStatus> BOOKING_DFA_DEFINITION = DFADefinition.builder(RECE)
-    .nonTerminalState(RECE).successorNodes(REJE, CANC, PENU, PENC, CONF)
-    .nonTerminalState(PENU).successorNodes(REJE, CANC, PENU, PENC)
-    .nonTerminalState(PENC).successorNodes(REJE, CANC, PENU, PENC, CONF)
-    .nonTerminalState(CONF).successorNodes(REJE, CMPL, PENU, PENC)
-    .terminalStates(CANC, CMPL, REJE)
+public class Booking extends AbstractStateMachine<String> implements Persistable<UUID> {
+
+  // TODO: validate this set since PENC has been broken into PENDING_UPDATES_CONFIRMATION and PENDING_AMENDMENTS_APPROVAL
+  private static final Set<String> CAN_BE_VALIDATED = Set.of(RECEIVED, PENDING_UPDATES_CONFIRMATION);
+
+//  private static final DFADefinition<BkgDocumentStatus> BOOKING_DFA_DEFINITION_OLD = DFADefinition.builder(RECE)
+//    .nonTerminalState(RECE).successorNodes(REJE, CANC, PENU, PENC, CONF)
+//    .nonTerminalState(PENU).successorNodes(REJE, CANC, PENU, PENC)
+//    .nonTerminalState(PENC).successorNodes(REJE, CANC, PENU, PENC, CONF)
+//    .nonTerminalState(CONF).successorNodes(REJE, CMPL, PENU, PENC)
+//    .terminalStates(CANC, CMPL, REJE)
+//    .build();
```

Comment:
Please remove this code; git will have it in history if we ever need it

Reply:
I'm planning to remove it as soon as I finish changes in the state engine (for those changes I think this will be handy to have  them side-by-side for quick comparison)